src/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(
+    #{$width-cell * $cells-in-row} + #{$gap-size * ($cells-in-row - 1)}
+  );
+  gap: $gap-size;
+
+  &__day {
+    height: $height-cell;
+    width: $width-cell;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    border: 1px solid black;
+    transition-duration: 0.5s;
```

Comment:
The `transition-duration` property should be accompanied by the `transition-property` property to specify which properties the transition effect should apply to. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)